src/test/java/com/google/TestCourseList.java

```diff
@@ -0,0 +1,38 @@
+package com.google;
+
+import static org.mockito.Mockito.*;
```

Comment:
wildcard imports are not allowed: https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports